service/src/test/java/com/ge/predix/acs/testutils/TestUtils.java

```diff
@@ -114,4 +115,17 @@ public MockMvcContext createWACWithCustomPOSTRequestBuilder(final WebApplication
         result.setMockMvc(MockMvcBuilders.webAppContextSetup(wac).defaultRequest(result.getBuilder()).build());
         return result;
     }
+
+    public static class TestAssertion {
```

Comment:
If you want to keep this inner class, I would put a `TODO` saying that we should start moving tests to use this `assertDoesNotThrow` method wherever we use the `try { ... } catch (Exception e) { Assert.fail(); }` pattern.

Reply:
I don't see it being used anywhere.  Should either use it or remove it.

Reply:
I have added a task in triage. in future we can implement this feature for exception handling